docker-compose.yml

```
@@ -43,6 +44,8 @@ services:
 volumes:
     data:
       name: our-volume
+    profile-images:
+      name: profile-images
```

Naming consistency,
starts with `our-` to recognize that this volume comes from `our` project 
```suggestion
    images:
      name: our-images
```